docs/api-spec.md

```diff
@@ -279,13 +280,16 @@ List responses have the following fields (based on [`meta.v1/ListMeta`](https://
 |--------------|------------|-------------|
 | `name`        | string    | REQUIRED    |
 | `description` | string    | REQUIRED    |
+| `type`      | Enum:<br>- `"string"` (default)<br>- `"array"` <br>- `"object"` | REQUIRED for `string` type <br> RECOMMENDED for `array` and `object` |
```

Comment:
"type" is new, so I think type has to be "recommended" for results. each of the supported values should be "recommended" as well.

Reply:
SG! Updated!